assets/xml/objects/object_shn.xml

```diff
@@ -64,22 +64,22 @@
-        <Limb Name="object_shn_Standardlimb_00E71C" Type="Standard" Offset="0xE71C" />
-        <Limb Name="object_shn_Standardlimb_00E728" Type="Standard" Offset="0xE728" />
-        <Limb Name="object_shn_Standardlimb_00E734" Type="Standard" Offset="0xE734" />
-        <Limb Name="object_shn_Standardlimb_00E740" Type="Standard" Offset="0xE740" />
-        <Limb Name="object_shn_Standardlimb_00E74C" Type="Standard" Offset="0xE74C" />
-        <Limb Name="object_shn_Standardlimb_00E758" Type="Standard" Offset="0xE758" />
-        <Limb Name="object_shn_Standardlimb_00E764" Type="Standard" Offset="0xE764" />
-        <Limb Name="object_shn_Standardlimb_00E770" Type="Standard" Offset="0xE770" />
-        <Limb Name="object_shn_Standardlimb_00E77C" Type="Standard" Offset="0xE77C" />
-        <Limb Name="object_shn_Standardlimb_00E788" Type="Standard" Offset="0xE788" />
-        <Skeleton Name="object_shn_Skel_00E7D0" Type="Flex" LimbType="Standard" Offset="0xE7D0" />
+        <Limb Name="object_shn_Standardlimb_00E6E0" Type="Standard" EnumName="SHN_LIMB_WAIST" Offset="0xE6E0" />
+        <Limb Name="object_shn_Standardlimb_00E6EC" Type="Standard" EnumName="SHN_LIMB_LEG_LEFT" Offset="0xE6EC" />
+        <Limb Name="object_shn_Standardlimb_00E6F8" Type="Standard" EnumName="SHN_LIMB_SHIN_LEFT" Offset="0xE6F8" />
+        <Limb Name="object_shn_Standardlimb_00E704" Type="Standard" EnumName="SHN_LIMB_FOOT_LEFT" Offset="0xE704" />
+        <Limb Name="object_shn_Standardlimb_00E710" Type="Standard" EnumName="SHN_LIMB_LEG_RIGHT" Offset="0xE710" />
+        <Limb Name="object_shn_Standardlimb_00E71C" Type="Standard" EnumName="SHN_LIMB_SHIN_RIGHT" Offset="0xE71C" />
+        <Limb Name="object_shn_Standardlimb_00E728" Type="Standard" EnumName="SHN_LIMB_FOOT_RIGHT" Offset="0xE728" />
+        <Limb Name="object_shn_Standardlimb_00E734" Type="Standard" EnumName="SHN_LIMB_TORSO" Offset="0xE734" />
+        <Limb Name="object_shn_Standardlimb_00E740" Type="Standard" EnumName="SHN_LIMB_SHOULDER_LEFT" Offset="0xE740" />
+        <Limb Name="object_shn_Standardlimb_00E74C" Type="Standard" EnumName="SHN_LIMB_FOREARM_LEFT" Offset="0xE74C" />
+        <Limb Name="object_shn_Standardlimb_00E758" Type="Standard" EnumName="SHN_LIMB_HAND_LEFT" Offset="0xE758" />
+        <Limb Name="object_shn_Standardlimb_00E764" Type="Standard" EnumName="SHN_LIMB_SHOULDER_RIGHT" Offset="0xE764" />
+        <Limb Name="object_shn_Standardlimb_00E770" Type="Standard" EnumName="SHN_LIMB_FOREARM_RIGHT" Offset="0xE770" />
+        <Limb Name="object_shn_Standardlimb_00E77C" Type="Standard" EnumName="SHN_LIMB_HAND_RIGHT" Offset="0xE77C" />
+        <Limb Name="object_shn_Standardlimb_00E788" Type="Standard" EnumName="SHN_LIMB_HEAD" Offset="0xE788" />
+        <Skeleton Name="object_shn_Skel_00E7D0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_SHN_LIMB_NONE" LimbMax="OBJECT_SHN_LIMB_MAX" EnumName="object_shn_Limbs" Offset="0xE7D0" />
```

Comment:
```suggestion
        <Skeleton Name="object_shn_Skel_00E7D0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_SHN_LIMB_NONE" LimbMax="OBJECT_SHN_LIMB_MAX" EnumName="ObjectShnLimbs" Offset="0xE7D0" />
```
Pascal case for name of the entire enum.

Reply:
Oh, I pulled this from the PR that Anghelo had opened for the automated, I can change it if that's what it needs to be

Reply:
Yeah, those are just autogenned named, and should be changed. In this case since you prefixed all the limb enums with `SHN_LIMB` I would go with 
```suggestion
        <Skeleton Name="object_shn_Skel_00E7D0" Type="Flex" LimbType="Standard" LimbNone="SHN_LIMB_NONE" LimbMax="SHN_LIMB_MAX" EnumName="ShnLimbs" Offset="0xE7D0" />
```